### PR TITLE
Change identification screen UI

### PR DESCRIPTION
**Pull request by Astach**
New identification UI, the new teaser text isn't included, because there's another ticket for it in progress and assigned to @Nexcius . I didn't want to redo what you might have already done :)
See the following attachement

new_ui

### CODE REVIEW

File: ui/src/main/res/drawable/schacc_rounded_container.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+            <solid android:color="#00CCCCCC" />
+
+            <corners android:radius="@dimen/schacc_container_radius" />
+        </shape>
+    </item>
+    <item>
+        <shape>
+            <padding android:top="1dp" />
+            <solid android:color="#10CCCCCC" />
```
Comment (Nexcius) [inline ui/src/main/res/drawable/schacc_rounded_container.xml]: Please specify this in colors.xml

File: ui/src/main/res/layout-h350dp-land/schacc_abstract_identification_fragment_layout.xml [hunk 2]
```diff
@@ -26,45 +27,61 @@
...
-        tools:layout_conversion_absoluteY="439dp" />
+            <TextView
+                android:id="@+id/identification_share_policy"
+                style="@style/schacc_text.secondary"
+                android:layout_marginTop="20dp"
+                android:drawableLeft="@drawable/schacc_ic_lock_outline"
+                android:drawablePadding="16dp"
+                android:drawableStart="@drawable/schacc_ic_lock_outline"
+                app:layout_constraintTop_toBottomOf="@+id/identification_input_view"
+                tools:layout_conversion_absoluteHeight="17dp"
```
Comment (Nexcius) [inline ui/src/main/res/layout-h350dp-land/schacc_abstract_identification_fragment_layout.xml]: All the `tools:layout_conversion` can be removed

File: ui/src/main/res/layout-land/schacc_abstract_identification_fragment_layout.xml [hunk 3]
```diff
@@ -26,41 +27,57 @@
...
-        android:layout_marginTop="16dp" />
+            <TextView
+                android:id="@+id/identification_share_policy"
+                style="@style/schacc_text.secondary"
+                android:layout_marginTop="20dp"
+                android:drawableLeft="@drawable/schacc_ic_lock_outline"
+                android:drawablePadding="16dp"
+                android:drawableStart="@drawable/schacc_ic_lock_outline"
+                app:layout_constraintTop_toBottomOf="@+id/identification_input_view"
+                tools:layout_conversion_absoluteHeight="17dp"
```
Comment (Nexcius) [inline ui/src/main/res/layout-land/schacc_abstract_identification_fragment_layout.xml]: Can be removed

File: ui/src/main/res/layout/schacc_abstract_identification_fragment_layout.xml [hunk 4]
```diff
@@ -25,45 +27,61 @@
...
-        tools:layout_conversion_absoluteY="439dp" />
+            <TextView
+                android:id="@+id/identification_share_policy"
+                style="@style/schacc_text.secondary"
+                android:layout_marginTop="20dp"
+                android:drawableLeft="@drawable/schacc_ic_lock_outline"
+                android:drawablePadding="16dp"
+                android:drawableStart="@drawable/schacc_ic_lock_outline"
+                app:layout_constraintTop_toBottomOf="@+id/identification_input_view"
+                tools:layout_conversion_absoluteHeight="17dp"
```
Comment (Nexcius) [inline ui/src/main/res/layout/schacc_abstract_identification_fragment_layout.xml]: Blast em' away, Scotty!

File: ui/src/main/res/layout/schacc_abstract_identification_fragment_layout.xml [hunk 5]
```diff
@@ -25,45 +27,61 @@
...
+
+            <com.schibsted.account.ui.ui.component.LoadingButton
+                android:id="@+id/identification_button_continue"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:text="@string/schacc_next_step"
+                android:layout_marginBottom="16dp"
+                tools:layout_conversion_absoluteHeight="48dp"
```
Comment (Nexcius) [inline ui/src/main/res/layout/schacc_abstract_identification_fragment_layout.xml]: Hulk smash!

File: ui/src/main/res/values/styles.xml [hunk 6]
```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (Nexcius) [inline ui/src/main/res/values/styles.xml]: There are no changes in this file

**Comment by Nexcius** [comment]
This is more of a change rather than an add, don't you think?

**Comment by Astach** [comment]
Done :1313

**Pull request merged**